src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -403,6 +403,36 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 		obj.paymentElement.mount( obj.selectors.paymentElement );
 	};
 
+	/**
+	 * Config for Stripe styling.
+	 *
+	 * @since TBD
+	 *
+	 * @return {Promise<void>}
+	 */
+	 obj.stripeAppearance = {
+		variables: {
+			borderRadius: '4px',
+			colorPrimary: '#334aff',
+			fontFamily: 'Helvetica Neue, Helvetica, -apple-system, BlinkMacSystemFont, Roboto, Arial, sans-serif',
+		},
+		rules: {
+			'.Tab': {
+				borderColor: '#d5d5d5',
+				boxShadow: 'none'
+			},
+			'.Tab--selected': {
+				borderWidth: '2px'
+			},
+			'.TabLabel': {
+				paddingTop: '6px'
+			},
+			'.Input': {
+				boxShadow: 'none'
+			}
+		}
```

Comment:
Pull that data from the PHP side look at the localize variable `tecTicketsCommerceGatewayStripeCheckout`